lti_consumer/tests/unit/plugin/test_views_lti_deep_linking.py

```diff
@@ -201,3 +202,81 @@ def test_dl_contents(self, has_access_patcher):  # pylint: disable=unused-argume
         self.assertEqual(resp.status_code, 200)
         expected_title = '{} | Deep Linking Contents'.format(self.lti_config.block.display_name)
         self.assertContains(resp, expected_title)
+
+    @ddt.data(
+        {'url': 'https://example.com'},
+        {'url': 'https://example.com', 'title': 'With Title'},
+        {'url': 'https://example.com', 'title': 'With Title', 'text': 'With Text'},
+        {
+            'url': 'https://example.com', 'title': 'With Title', 'text': 'With Text',
+            'icon': {'url': 'https://link.to.icon', 'width': '20px', 'height': '20px'},
+        },
+        {
+            'url': 'https://example.com', 'title': 'With Title', 'text': 'With Text',
+            'thumbnail': {'url': 'https://link.to.thumbnail', 'width': '20px', 'height': '20px'},
+        },
+        {
+            'url': 'https://example.com', 'title': 'With Title', 'text': 'With Text',
+            'window': {'targetName': 'newWindow', 'windowFeatures': 'width=200px,height=200px'},
+        },
```

Comment:
can we test the cases for iframe and embed as well ?

Reply:
``iframe`` and ``embed`` has not been implemented. I'm not sure what are those in this ``link`` context.

Reply:
okay may be we need more clarification on this but other than that it's okay